mobile/app/(app)/tosign/index.tsx

```diff
@@ -39,47 +39,36 @@ export default function Page() {
   const remote = useAppSelector(selectRemote)
   const [signedTx, setSignedTx] = useState<string | undefined>(undefined)
   const [gasWanted, setGasWanted] = useState<bigint>(BigInt(0))
-  // const session = useAppSelector(selectSession);
-  // const sessionWanted = useAppSelector(selectSessionWanted);
 
   console.log('txInput', txInput)
   console.log('bech32Address', bech32Address)
   console.log('clientName', clientName)
   console.log('reason', reason)
-  // console.log('session', session);
-  // console.log('sessionWanted', sessionWanted);
-
-  // useEffect(() => {
-  //   if (session) {
-  //     // if we have a session, mwe can sign the tx and return to the requester.
-  //     setTimeout(() => {
-  //       signTxAndReturnToRequester()
-  //     }, 300);
-  //   }
-  // }, [session])
 
   useEffect(() => {
     ;(async () => {
-      if (!chainId || !remote) throw new Error('No chainId or remote found.')
+      if (!chainId || !remote || !bech32Address) return
       gnonative.setChainID(chainId)
       gnonative.setRemote(remote)
+      gnonative.activateAccount(bech32Address)
```

Comment:
Does this need await? Note that activateAccount uses the Keybase DB. It needs to finish before any other use of the Keybase DB.